fox_consumer/consumer_db_support.py

```diff
@@ -0,0 +1,56 @@
+import psycopg2
+
+class FoxConsumerPostgres(object):
+    def __init__(self):
+        """
+        This function will initialize postgress database connection.
+        it will connect to "mtdb" database and check if the "fox_item" table exist, id not it will create it.
+        """
+        # Connect to an existing database
+        self.postgress_connection = psycopg2.connect("dbname=mydb user=aslishemesh")
+        # Open a cursor to perform database operations
+        self.db_curser = self.postgress_connection.cursor()
+
+        # verify if the table already exist
+        sql_query_table = "select exists(select * from information_schema.tables where table_name='fox_items');"
+        self.db_curser.execute(sql_query_table)
+        table_exist = self.db_curser.fetchone()[0]
+        if table_exist == False:
```

Comment:
You can write `if not table_exist:`, you don't need to explicitly check that its False